src/components/interactive-builder/edit-question.modal.tsx

```diff
@@ -397,57 +401,63 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
-              {questionTypes.map((questionType, key) => (
-                <SelectItem text={questionType} value={questionType} key={key} />
-              ))}
-            </Select>
-            <Select
-              defaultValue={questionToEdit.questionOptions.rendering}
-              onChange={(event: React.ChangeEvent<HTMLSelectElement>) => setFieldType(event.target.value as RenderType)}
-              id="renderingType"
-              invalidText={t('validFieldTypeRequired', 'A valid field type value is required')}
-              labelText={t('renderingType', 'Rendering type')}
-              required
-            >
-              {!fieldType && <SelectItem text={t('chooseRenderingType', 'Choose a rendering type')} value="" />}
-              {fieldTypes.map((fieldType, key) => (
-                <SelectItem text={fieldType} value={fieldType} key={key} />
-              ))}
-            </Select>
+            {'label' in questionToEdit && (
+              <>
+                <RadioButtonGroup
+                  defaultSelected={/true/.test(questionToEdit?.required?.toString()) ? 'required' : 'optional'}
+                  name="isQuestionRequired"
+                  legendText={t(
+                    'isQuestionRequiredOrOptional',
+                    'Is this question a required or optional field? Required fields must be answered before the form can be submitted.',
+                  )}
+                >
+                  <RadioButton
+                    id="questionIsNotRequired"
+                    defaultChecked={true}
+                    labelText={t('optional', 'Optional')}
+                    onClick={() => setIsQuestionRequired(false)}
+                    value="optional"
+                  />
+                  <RadioButton
+                    id="questionIsRequired"
+                    defaultChecked={false}
+                    labelText={t('required', 'Required')}
+                    onClick={() => setIsQuestionRequired(true)}
+                    value="required"
+                  />
+                </RadioButtonGroup>
+                <Select
+                  defaultValue={questionToEdit.type}
+                  onChange={(event: React.ChangeEvent<HTMLSelectElement>) =>
+                    setQuestionType(event.target.value as QuestionType)
+                  }
+                  id={'questionType'}
+                  invalidText={t('typeRequired', 'Type is required')}
+                  labelText={t('questionType', 'Question type')}
+                  required
+                >
+                  {!questionType && <SelectItem text={t('chooseQuestionType', 'Choose a question type')} value="" />}
+                  {questionTypes.map((questionType, key) => (
+                    <SelectItem text={questionType} value={questionType} key={key} />
+                  ))}
+                </Select>
+              </>
+            )}
+            {!questionToEdit.value && 
```

Comment:
Why are we conditionally displaying the 'renderingType' field only when it has no value? If the intent is to prevent users from modifying the rendering type, we don't need to do that. Let's always display this field regardless.